pkg/defaults/defaults.go

```diff
@@ -136,6 +136,14 @@ func FromConfig(
 			if _, ok := requiredNames[string(rawStep.OutputImageTagStepConfiguration.From)]; ok || !rawStep.OutputImageTagStepConfiguration.Optional {
 				stepLinks = append(stepLinks, step.Creates()...)
 			}
+		} else if rawStep.PrePublishOutputImageTagStepConfiguration != nil {
```

Comment:
where you pass this step? This will always be nil, right?

Reply:
No, it's a raw step, not in the top level config currently since it's still new and not something we want everyone to be using necessarily.

Reply:
See https://github.com/chancez/ci-operator/blob/pre_publish/pkg/defaults/defaults.go#L440

Reply:
Did a quick view. I didnt notice it, thanks